MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -1515,6 +1519,7 @@
 				C561735A1F182113005954F6 /* RouteStep.swift */,
 				351927351F0FA072003A702D /* ScreenCapture.swift */,
 				3A8187C824BDAE9C00708F19 /* URLSession.swift */,
+				11D1F89F2696048D0053A93F /* Dictionary.swift */,
```

Comment:
This file contains only an extension on Dictionary, but from outside it seems like a new ``Dictionary`` type. On Apple platforms there is a non-formal convention to append something to the file name. For example:
- ``Dictionary++``
- ``Dictionary+Extensions``
- ``Dictionary+[GroupName]``

This approach is already used in our repository.